CodeIgniter/application/views/templates/header.php

```diff
@@ -0,0 +1,7 @@
+<html>
+	<head>
+		<title>Admin Page</title>
+	</head>
```

Comment:
View/includes has a header and footer with a CSS called and a menu bar already implemented